### DISCUSSION TITLE
Change color for "Request Changes"

### DISCUSSION THREAD

**Discussion by codie3611** [community/community#7731] [Pull Requests]
Requesting changes as a reviewer during PRs is using a red color. Psychological this feels "mean", driving me away of using it since it seems quite offensive. On the other side as as an author it makes me feel attacked or generally not feeling good. Changing this to a less offensive color such as blue makes this feature much more attractive, especially since requesting changes are also suggestions and suggestions are rather blue.

**Comment by Andre601** [comment]
For me personally is the red colour not really "offensive" or "mean".
If the requested change would be required to allow the PR to be merged can the red colour be seen as a "This is important! You have to change something".

What I dislike more is the fact that non-maintainers seem to have a different, more neutral colour (grey) when they themself submit a review with change requests...
It (unintentionally) makes their reviews feel a lot less important, even tho they could also point out critical changes that need to be addressed.

My personal suggestion is to make the change request have two colours used in general: Grey for optional (not required) changes and red for required changes.
That way could a maintainer or other review point out changes that NEED to be changed and those that would be nice to change but aren't necessary right now.

**Reply by codie3611** [reply]
Appreciate your understanding of the color red and interesting suggestion. Errors and the color red are just bundled in my head. How about the following synthesis: Binding the color to important and unimportant changes as a task list.

- [ ] Important Todo <-- Should be red
- [ ] Unimportant Todo <-- Should be grey

If there is an important todo present, a red color is used generally and unimportant todos are marked grey and similarly to Bitbucket, the author may not merge until all important todos are done or have been removed again. My only fear of this approach is the implicit complexity of different todo types and their effect on coloring may confuse users. The benefit though is, that one could leave the default behavior red unless the new todo items are used, avoiding some confusion when introducing this feature.

**Reply by Andre601** [reply]
Could be an interesting solution, but the obvious difficulty would be to find a way of making this easy to use/implement, preferably with a simple markdown format, just like f.e. change suggestions in reviews.

**Comment by glaebhoerl** [comment]
GitHub already [changed](https://github.com/github/roadmap/issues/289) the color for closed issues, citing some similar reasoning:

> ... and the general scariness of seeing red ...

So I hope that adds some motivation here as well.
